<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TabCPU</class>
 <widget class="QWidget" name="TabCPU">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>712</width>
    <height>189</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="layoutDirection">
   <enum>Qt::LeftToRight</enum>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <property name="leftMargin">
    <number>13</number>
   </property>
   <property name="topMargin">
    <number>25</number>
   </property>
   <property name="rightMargin">
    <number>13</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <layout class="QVBoxLayout" name="CPUMainVLayout">
     <property name="spacing">
      <number>6</number>
     </property>
     <item>
      <layout class="QGridLayout" name="CPUgridLayout">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <item row="4" column="1">
        <widget class="QLabel" name="label_irq">
         <property name="toolTip">
          <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Time spent for servicing hardware interrupts&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
         <property name="text">
          <string>Irq:</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item row="1" column="6" alignment="Qt::AlignRight|Qt::AlignVCenter">
        <widget class="QLabel" name="label_threads">
         <property name="text">
          <string>Threads:</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item row="2" column="4">
        <widget class="QLabel" name="label_system_unit">
         <property name="text">
          <string>%</string>
         </property>
        </widget>
       </item>
       <item row="3" column="5" alignment="Qt::AlignLeft">
        <widget class="ColorButton" name="color_picker_nice_value" native="true"/>
       </item>
       <item row="1" column="8">
        <widget class="QLabel" name="label_threads_value">
         <property name="styleSheet">
          <string notr="true"/>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QLabel" name="label_user">
         <property name="toolTip">
          <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Time spent by normal processes executing in user mode&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
         <property name="text">
          <string>User:</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item row="2" column="3">
        <widget class="QLabel" name="label_system_value">
         <property name="toolTip">
          <string/>
         </property>
         <property name="styleSheet">
          <string notr="true"/>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QLabel" name="label_nice">
         <property name="toolTip">
          <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Time spent by niced (prioritized) processes executing in
           user mode&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;
          </string>
         </property>
         <property name="text">
          <string>Nice:</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item row="3" column="4">
        <widget class="QLabel" name="label_nice_unit">
         <property name="text">
          <string>%</string>
         </property>
        </widget>
       </item>
       <item row="5" column="5" alignment="Qt::AlignLeft">
        <widget class="ColorButton" name="color_picker_idle_value" native="true"/>
       </item>
       <item row="2" column="1">
        <widget class="QLabel" name="label_system">
         <property name="toolTip">
          <string comment="toolTip information" extracomment="That is HTML format in case of formating needed">&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Time
           spent by processes executing in kernel mode.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;
          </string>
         </property>
         <property name="text">
          <string>System:</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item row="1" column="10" rowspan="4">
        <widget class="CPUGraphBar" name="widget_graph" native="true">
         <property name="sizePolicy">
          <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
        </widget>
       </item>
       <item row="0" column="5">
        <spacer name="horizontalSpacer_1">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::MinimumExpanding</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="3" column="3">
        <widget class="QLabel" name="label_nice_value">
         <property name="toolTip">
          <string/>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item row="1" column="3">
        <widget class="QLabel" name="label_user_value">
         <property name="toolTip">
          <string/>
         </property>
         <property name="styleSheet">
          <string notr="true">QToolTip { color: #ffffff; border: 0px; }</string>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item row="0" column="10" alignment="Qt::AlignHCenter">
        <widget class="QLabel" name="label_cpu_usage">
         <property name="text">
          <string>CPU Usage</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item row="4" column="5" alignment="Qt::AlignLeft">
        <widget class="ColorButton" name="color_picker_irq_value" native="true"/>
       </item>
       <item row="1" column="4">
        <widget class="QLabel" name="label_user_unit">
         <property name="text">
          <string>%</string>
         </property>
        </widget>
       </item>
       <item row="5" column="3">
        <widget class="QLabel" name="label_idle_value">
         <property name="toolTip">
          <string/>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item row="5" column="4">
        <widget class="QLabel" name="label_idle_unit">
         <property name="text">
          <string>%</string>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <spacer name="horizontalSpacer_4">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="5" column="1">
        <widget class="QLabel" name="label_idle">
         <property name="toolTip">
          <string>Time spent doing nothing</string>
         </property>
         <property name="text">
          <string>Idle:</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item row="2" column="8">
        <widget class="QLabel" name="label_processes_value">
         <property name="styleSheet">
          <string notr="true"/>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item row="0" column="2">
        <spacer name="horizontalSpacer_6">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Preferred</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="1" column="5" alignment="Qt::AlignLeft">
        <widget class="ColorButton" name="color_picker_user_value" native="true"/>
       </item>
       <item row="4" column="4">
        <widget class="QLabel" name="label_irq_unit">
         <property name="text">
          <string>%</string>
         </property>
        </widget>
       </item>
       <item row="2" column="6" alignment="Qt::AlignRight|Qt::AlignVCenter">
        <widget class="QLabel" name="label_processes">
         <property name="text">
          <string>Processes:</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item row="0" column="9">
        <spacer name="horizontalSpacer_3">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::MinimumExpanding</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="0" column="7">
        <spacer name="horizontalSpacer_7">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Preferred</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="4" column="3">
        <widget class="QLabel" name="label_irq_value">
         <property name="toolTip">
          <string/>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item row="0" column="11">
        <spacer name="horizontalSpacer_5">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="2" column="5" alignment="Qt::AlignLeft">
        <widget class="ColorButton" name="color_picker_system_value" native="true"/>
       </item>
      </layout>
     </item>
     <item>
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ColorButton</class>
   <extends>QWidget</extends>
   <header>.widget_color_pickup</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>CPUGraphBar</class>
   <extends>QWidget</extends>
   <header>.widget_cpugraphbar</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
